script.js

```diff
@@ -0,0 +1,116 @@
+const inputJs = document.querySelector(".input"),
+    toDoNewJs = inputJs.querySelector("#todo-new"),
+    listWaitingJs = document.querySelector(".waiting"),
+    listCompleteJs = document.querySelector(".complete");
+
+const TODOS = "toDos"; //key for saving todos in local storage
+
+let toDos = [];
+let maxId = 0;
+let countWaiting = 0, countComplete = 0;
+let tmpType; //only for deleteToDo and moveToDo
+
+function saveToDos() {
+    localStorage.setItem(TODOS, JSON.stringify(toDos));
+}
+
+function printCount() {
+    const countWaitingJs = document.querySelector("#count-waiting"),
+        countCompleteJs = document.querySelector("#count-complete");
+    countWaitingJs.innerText = countWaiting;
+    countCompleteJs.innerText = countComplete;
+}
+
+function deleteToDo(event){
+    const li = event.target.parentNode;
+    tmpType = li.parentNode.className;
+    
+    if(tmpType === "list waiting"){ 
+        listWaitingJs.removeChild(li);
+        countWaiting -= 1; 
+    } 
+    else{
+        listCompleteJs.removeChild(li);
+        countComplete -= 1;    
+    }
+    printCount();
+   
+    const cleanToDos = toDos.filter(function(toDo){
+        return toDo.id !== parseInt(li.id);
+    });
+    toDos = cleanToDos;
+    saveToDos();
+
+    maxId += 1; //prevent ovelapping of id
+}
+
+function moveToDo(event){
+    deleteToDo(event);
+    const targetContent = event.target.innerText;
+    if(tmpType === "list waiting") { printToDo(targetContent, "comp"); }
+    else { printToDo(targetContent, "wait");}
+}
+
+function printToDo(content, type) {
+    const li = document.createElement("li");
+    li.setAttribute("class", "todo");
```

Comment:
`classList.add()`를 사용하면 좀 더 간결해질 것 같아요~ (아래 코드들도 마찬가지)
```suggestion
    li.classList.add("todo");
```

Reply:
네! 메서드 공부 좀 더 해야겠어요 ^-ㅜ

Reply:
```suggestion
    li.class = "todo";
```
저는 이렇게 했었어요 ㅎㅎ